cluster-scope/base/user.openshift.io/groups/octo-et-security/group.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+  name: octo-et-security
```

Comment:
This group name doesn't match what is requested in the [support issue](https://github.com/operate-first/support/issues/628), is that ok @durandom  and @lukehinds?

Reply:
Yeah, I came up with a better, generic group name :)